Woof/Woof/ViewModel/SitterProfileViewModel.swift

```diff
@@ -55,6 +81,16 @@ final class SitterProfileViewModel: ObservableObject {
 
     private lazy var currentSitter: Sitter = loadSitterFromStorage()
 
+    private func upload(_ sitter: Sitter) async throws {
+        let endpoint = WoofAppEndpoint.addNewSitter(sitter.asDictionary())
+        _ = try await NetworkService<WoofAppEndpoint>().request(endpoint)
+    }
+
+    private func handleError(_ error: AppError) {
```

Comment:
I think it's appropriate for this function.
```suggestion
    @MainActor private func handleError(_ error: AppError) {
```

Reply:
Fixed with 7282f12